vendor/data/codelists/BEIS/benefitting_countries.yml

```diff
@@ -4,2009 +4,1141 @@ data:
     recipient_code: "130"
     code: DZ
     regions:
-      - name: Africa, regional
-        code: "298"
-        level: 1
-      - name: North of Sahara, regional
-        code: "189"
-        level: 2
+      - "298"
```

Comment:
And now the ordering of the regions in the array is less obvious, if I put the two codes in the wrong order here, things will not work as expected.